src/lib/callApi.ts

```diff
@@ -0,0 +1,39 @@
+import { dev } from '$app/environment';
+import { get } from 'svelte/store';
+import { xsrf } from './stores';
+
+const callApi = async <
+  ResBody = Record<string, unknown>,
+  ReqBody = Record<string, unknown>
+>(
```

Comment:
This should be generic. But we could do that later I guess, when we actually add functions to perform api requests :) 

Just don't wait too long cause it's gonna be a chore then.

(We should really generate and use openAPI schemas from the backend)